From 279239c4c670edbde12345aca4fadb7f07d503e8 Mon Sep 17 00:00:00 2001
From: devtools-ci-autoroll-builder
 <devtools-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Date: Thu, 17 Oct 2024 05:07:12 -0700
Subject: [PATCH 01/13] Update DevTools DEPS (trusted)

Rolling build: https://chromium.googlesource.com/chromium/src/build/+log/93926da..354468d

Roll created at https://cr-buildbucket.appspot.com/build/8733831322268823409

Bug: none
Change-Id: Ib85e8cdbdec9870f5277cd4fb053adb909739d5d
Reviewed-on: https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/5938688
Commit-Queue: Devtools Autoroller <devtools-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
Bot-Commit: Devtools Autoroller <devtools-ci-autoroll-builder@chops-service-accounts.iam.gserviceaccount.com>
---
 DEPS  | 2 +-
 build | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/DEPS b/DEPS
index de275554a8..9b1636f0c6 100644
--- a/DEPS
+++ b/DEPS
@@ -12,7 +12,7 @@ vars = {
   'build_with_chromium': False,
 
   'build_url': 'https://chromium.googlesource.com/chromium/src/build.git',
-  'build_revision': '93926da1cd27aa9e3c7fe3650d9c0a2ca2373c30',
+  'build_revision': '354468d2ec3b0700a6812f2ec596b730bcd6bfd3',
 
   'buildtools_url': 'https://chromium.googlesource.com/chromium/src/buildtools.git',
   'buildtools_revision': 'd1850bb09e7916134a5364ede6025a62c34ad113',
diff --git a/build b/build
index 93926da1cd..354468d2ec 160000
--- a/build
+++ b/build
@@ -1 +1 @@
-Subproject commit 93926da1cd27aa9e3c7fe3650d9c0a2ca2373c30
+Subproject commit 354468d2ec3b0700a6812f2ec596b730bcd6bfd3
-- 
2.39.5 (Apple Git-154)

